content-api/content-actors/src/main/scala/org/sunbird/content/actors/EventActor.scala

```diff
@@ -174,4 +175,33 @@ class EventActor @Inject()(implicit oec: OntologyEngineContext, ss: StorageServi
       })
     }).flatMap(f => f)
   }
+
+  override def systemUpdate(request: Request): Future[Response] = {
+    RedisCache.delete(request.get("identifier").asInstanceOf[String])
```

Comment:
perform the redis delete operation as the last operation in this method when API is successful.